### PR TITLE
[Refactor:Developer] Remove key_to_click from buttons

### PR DESCRIPTION
**Pull request by micpap25**
Per Shail from #7819, remove all uses of `key_to_click` on buttons, where this class is redundant and serves no purpose since the functionality that `key_to_click` implements is the default for buttons.

### CODE REVIEW

File: site/app/templates/grading/electronic/AutogradingPanel.twig [hunk 1]
```diff
@@ -2,8 +2,8 @@
 <div id="autograding_results" class="rubric_panel" data-highest-version="{{ highest_version }}" data-gradeable-id="{{ gradeable_id }}" data-user-id="{{ user_id }}" data-testid="autograding-results">
     <div>
         <span class="grading_label">Autograding Testcases</span>
-        <button id="autograding-results-open-all" class="btn btn-default key_to_click" tabindex="0">Expand All</button>
-        <button id="autograding-results-close-all" class="btn btn-default key_to_click" tabindex="0">Close All</button>
```
Comment (powe97) [inline site/app/templates/grading/electronic/AutogradingPanel.twig:6]: Remove the trailing space here. Looks good otherwise.

**Review by powe97 — CHANGES_REQUESTED** [review]

File: site/app/templates/grading/electronic/AutogradingPanel.twig [hunk 2]
```diff
@@ -2,8 +2,8 @@
...
+        <button id="autograding-results-open-all" class="btn btn-default" tabindex="0">Expand All</button>
+        <button id="autograding-results-close-all" class="btn btn-default " tabindex="0">Close All</button>
```
Comment (powe97) [inline site/app/templates/grading/electronic/AutogradingPanel.twig:6]:
```suggestion
        <button id="autograding-results-close-all" class="btn btn-default" tabindex="0">Close All</button>
```

**Review by powe97 — CHANGES_REQUESTED** [review]

File: site/app/templates/grading/electronic/AutogradingPanel.twig (hunk 2 above)
Comment (powe97) [reply site/app/templates/grading/electronic/AutogradingPanel.twig:6]: Looks good otherwise.

**Review by ziesski — APPROVED** [review]
Would have been nice if you cherry pick merged my commit, but its fine. The code changes seem reasonable to me. 

There should be a follow-up PR to address the issue with anchor tag, but it would have some stylistic changes, so we would need to review with care.

**Comment by micpap25** [comment]
> Would have been nice if you cherry pick merged my commit, but its fine. The code changes seem reasonable to me.

Didn't realize you could do this cross-PR, sorry!

**Comment by bmcutler** [comment]
@micpap25 
Please fix version conflicts

**Pull request merged**